.vib/prestashop/cypress/cypress/integration/prestashop_spec.js

```diff
@@ -0,0 +1,102 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+// Note: Interaction with the UI is preferred over cy.visit() as
+// PrestaShop requires to provide a token within the URL or else
+// a security warning page is shown.
+
+it('allows login in and out', () => {
+  cy.login();
+  cy.contains('Activity overview');
+
+  cy.get('#header_employee_box').click();
+  cy.contains('Sign out').click();
+  cy.get('form#login_form');
+});
+
+it('allows registering a new product', () => {
+  cy.login();
+  cy.contains('a[href*="catalog/products"]', 'Catalog').click();
+  cy.contains('a[href*="catalog/products"]', 'Products').click();
+  cy.get('div.header-toolbar').within(() => {
+    cy.contains('a[href*="catalog/products/new"]', 'New product').click();
+  });
+
+  cy.fixture('products').then((products) => {
+    cy.get('input[placeholder="Enter your product name"]').type(
+      `${products.newProduct.name} ${random}`
+    );
+    cy.get('input[type="file"][accept="image/*"]').selectFile(
+      'cypress/fixtures/images/product_image.png',
+      { force: true }
+    );
+    cy.get('input#form_step1_price_shortcut').type(products.newProduct.price);
+    cy.contains('input#submit', 'Save').click();
+    cy.contains('Settings updated');
+
+    cy.contains('a[href*="catalog/products"]', 'Products').click();
+    cy.contains(`${products.newProduct.name} ${random}`);
+  });
+});
+
+it('allows registering a new customer', () => {
+  cy.login();
+  cy.contains('a[href*="sell/customers"]', 'Customers').click();
+  cy.contains('li#subtab-AdminCustomers', 'Customers').click();
+  cy.get('div.header-toolbar').within(() => {
+    cy.contains('a[href*="customers/new"]', 'Add new customer').click();
+  });
+
+  cy.fixture('customers').then((customers) => {
+    cy.get('input#customer_first_name').type(customers.newCustomer.firstName);
```

Comment:
Should we randomise the data for adding customers or is it's OK to duplicate data?

Reply:
It is OK (and so it is from a conceptual point of view) for two or more customers to have the same name. We are however randomizing the email, and it is in fact the element we later check to ensure the test has succeeded.

Reply:
OK, perfect by me!